src/index.js

```diff
@@ -54,34 +54,42 @@ function matches (node, query) {
   return (node.matches || node.msMatchesSelector).call(node, query);
 }
 
-function nodeFromHtml (html) {
-  const div = document.createElement('div');
-  div.innerHTML = html;
-  return div.firstElementChild;
+function getInstantiatedNodeWithinFixture(node, isRootNode) {
+  const isStringNode = typeof node === 'string';
+
+  // If the fixture has been removed from the document, re-insert it.
+  if (!body.contains(fixture)) {
+    body.appendChild(fixture);
+  }
+
+  if (isRootNode) {
+    setFixtureContent( node, isStringNode );
+  }
+
+  return isStringNode
+    ? fixture.firstElementChild
+    : node;
+}
+
+function setFixtureContent( node, shouldSetChildrenViaString ) {
```

Comment:
about semistandard. I know it's installed via `skate/build` package, but, if user want's to fix lint errors, `sk-lint -- --fix` just doesn't work and user has to manually install semistandard to run those fixes ( it's not very convenient to fix those manually ). Maybe we should create `sk-lint-fix` command withing `skate/build` package and also add git pre commit hooks which will run tests/lint so we don't have to deal with these kind of problems during PRs. WDYT?